watertap/unit_models/zero_order/nanofiltration_zo.py

```diff
@@ -72,3 +72,11 @@ def load_parameters_from_database(self, use_default_removal=False):
         # TODO: Do not have parameters for these yet - need to add to datafile
         self.deltaP_treated.fix(0)
         self.deltaP_byproduct.fix(0)
+
+    def _get_performance_contents(self, time_point=0):
+        perf_dict = super()._get_performance_contents(time_point=0)
```

Comment:
Should this just be `time_point` instead of `time_point=0`?

Reply:
No, we want it to default to time=0 so that people doing steady-state models (most of them) don't need to worry about this argument. This is only there to support dynamic cases (and even then only allows you to pick a single time point).

Reply:
Right, setting the default for time_point to 0 supports steady-state models so users don't have to worry about setting this unless they're working with dynamic models--but I was referring to 
`perf_dict = super()._get_performance_contents(time_point=0)`

I figured since time_point is already set to 0 in the function call, you don't need this, and if the user wanted some other time_point, say, time_point=2, I would think we'd want to call super on that same time_point of 2, not 0. Or are you saying that regardless of the time_point of interest, `super()._get_performance_contents` should always use `time_point=0` for whatever reason?

Reply:
Ah - missed that. I saw the first line in the GitHub snippet and not the second.